src/osemgrep/cli_ci/Git_metadata.ml

```diff
@@ -0,0 +1,171 @@
+(*****************************************************************************)
+(* Types and constants *)
+(*****************************************************************************)
+
+type extension = unit
+
+type env = {
+  _SEMGREP_REPO_NAME : string option;
```

Comment:
smart, nice way to match the env variable by prefixing with a _.
I'll do that too maybe in Semgrep_envvars.mli then!

Reply:
Yes, it's a question of style that one may not like 🙂